### PR TITLE
Update Changelog

### PR DESCRIPTION
**Pull request by fgiudici**
$> auto-changelog

### CODE REVIEW

**Review by davidcassany — COMMENTED** [review]
I am far from being a fan of this 😅 but still I have no strong opinion, I can live with it if other like it.

I don't like it mainly for two reasons:

* We can't really check these logs are not customized manually (so including errors) or if they have been created with a different criteria of previous updates. So I find it error prone.
* If you checkout v0.6.3 you don't get the changelog for v0.6.3 tag...

I can't think of a nice workaround for the above, probably there is though.

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,51 @@ All notable changes to this project will be documented in this file. Dates are d
...
+- Add OBS workflow to update elemental-operator package [`#200`](https://github.com/rancher/elemental-operator/pull/200)
+- Add vendor for obs integration [`#198`](https://github.com/rancher/elemental-operator/pull/198)
+- release: enhance release pipeline [`#195`](https://github.com/rancher/elemental-operator/pull/195)
+- operator: drop duplicated import of elemental APIs [`#199`](https://github.com/rancher/elemental-operator/pull/199)
+- Disable CGO under arm for register binaries + restore SBOM [`#193`](https://github.com/rancher/elemental-operator/pull/193)
+- Revert "Add sbom to releases and attach to containers" [`#191`](https://github.com/rancher/elemental-operator/pull/191)
+- Add elemental GlobalRole for Rancher UI [`#187`](https://github.com/rancher/elemental-operator/pull/187)
+- Add reasons for conditions [`#185`](https://github.com/rancher/elemental-operator/pull/185)
+- lint: dont overshadow var [`#172`](https://github.com/rancher/elemental-operator/pull/172)
+- Add v0.6.0 changelog [`#182`](https://github.com/rancher/elemental-operator/pull/182)
```
[the remark this answers is not in the document]
Comment (fgiudici) [reply CHANGELOG.md:49]: lol, you are right 🤣 
my point on the CHANGELOG is that we either keep it up to date or we get rid of it.
It is also not perfect at all, I would just put the PR list and for sure drop things like " Add v0.6.0 changelog": but having it auto-generated by just calling `auto-changelog` is so handy, that I would tolerate weird stuff here and there to keep the process (almost) effortless (`auto-changelog` regenerates the whole CHANGELOG each time).

[the remark this answers is not in the document]
Comment (davidcassany) [reply CHANGELOG.md:49]: > my point on the CHANGELOG is that we either keep it up to date or we get rid of it.

sure, a decision needs to be taken on how to handle this, and probably extend it to the other repositories. Probably we should include a minimal `RELEASING.md` file including few words over the tagging and versioning process and criteria.

[the remark this answers is not in the document]
Comment (fgiudici) [reply CHANGELOG.md:49]: Makes sense to me. Closing this PR for now, we should first discuss how/what to release.

**Pull request closed without merging**